README.md

```diff
@@ -24,42 +24,8 @@ The figure below illustrates the software flow through the architecture componen
 5. **Publish Inference Events/Data:** Analytics inferences are formatted and passed to the destination message bus specified in the request.
 6.  **Export Data:** Publish prepared (transformed, enriched, filtered, etc.) and groomed (formatted, compressed, encrypted, etc.) data to external systems (be it analytics package, enterprise or on-premises application, cloud systems like Azure IoT, AWS IoT, or Google IoT Core, etc.
 
-
```

Comment:
Isn't the above Overview and How it works in the edgex-doc documentation? If not it should be and removed for here.

Reply:
It is - I had just thought some basic info left on this page might be nice to leave, but I can remove it.

Reply:
Sounds good